diff --git a/Makefile b/Makefile.new
index 6923a07..9abe036 100644
--- a/Makefile
+++ b/Makefile.new
@@ -125,7 +125,7 @@ AAREDIR= libapparmor_re
 AAREOBJECT = ${AAREDIR}/libapparmor_re.a
 AAREOBJECTS = $(AAREOBJECT)
 AARE_LDFLAGS = -static-libgcc -static-libstdc++ -L. $(LDFLAGS)
-AALIB = -Wl,-Bstatic -lapparmor -Wl,-Bdynamic -lpthread
+AALIB = -Wl,-Bdynamic -lapparmor -Wl,-Bdynamic -lpthread

 ifdef WITH_LIBINTL
 	AALIB += -lintl
