--- a/Makefile
+++ b/Makefile
@@ -440,9 +440,9 @@ install-arch: $(INSTALLDEPS)
 install-indep: indep
 	install -m 755 -d $(INSTALL_CONFDIR)
 	install -m 644 parser.conf $(INSTALL_CONFDIR)
-	install -m 755 -d $(APPARMOR_BIN_PREFIX)
-	install -m 755 rc.apparmor.functions $(APPARMOR_BIN_PREFIX)
-	install -m 755 profile-load $(APPARMOR_BIN_PREFIX)
+	install -m 755 -d ${DESTDIR}/usr/libexec
+	install -m 755 profile-load ${DESTDIR}/usr/libexec
+	install -m 755 rc.apparmor.functions ${DESTDIR}/usr/libexec
 	$(MAKE) -C po install NAME=${NAME} DESTDIR=${DESTDIR}
 	$(MAKE) install_manpages DESTDIR=${DESTDIR}

