diff --git a/pkg/ffmpeg/config_components.h b/pkg/ffmpeg/config_components.h
index 60dc2985..85777349 100644
--- a/pkg/ffmpeg/config_components.h
+++ b/pkg/ffmpeg/config_components.h
@@ -451,11 +451,11 @@
 #define CONFIG_PCM_S8_PLANAR_DECODER 0
 #define CONFIG_PCM_S16BE_DECODER 0
 #define CONFIG_PCM_S16BE_PLANAR_DECODER 0
-#define CONFIG_PCM_S16LE_DECODER 0
+#define CONFIG_PCM_S16LE_DECODER 1
 #define CONFIG_PCM_S16LE_PLANAR_DECODER 0
 #define CONFIG_PCM_S24BE_DECODER 0
 #define CONFIG_PCM_S24DAUD_DECODER 0
-#define CONFIG_PCM_S24LE_DECODER 1
+#define CONFIG_PCM_S24LE_DECODER 0
 #define CONFIG_PCM_S24LE_PLANAR_DECODER 0
 #define CONFIG_PCM_S32BE_DECODER 0
 #define CONFIG_PCM_S32LE_DECODER 0
