diff --git a/Makefile b/Makefile
index b82941b..c459e3e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,6 @@
 CC := gcc
 CFLAGS := -O2 -Wall -Werror -Wno-error=unused-result -pedantic -std=gnu99
+DESTDIR := /usr/local
 INSTALL := install -Dm755

 .PHONY: clean install
@@ -8,16 +9,16 @@ hummingbird: src/signal.c src/shutdown.c src/init.c src/hummingbird.c
 	$(CC) $(CFLAGS) src/signal.c src/shutdown.c src/init.c src/hummingbird.c -o hummingbird

 install: hummingbird
-	mkdir -p "$(DESTDIR)/usr/lib/hummingbird"
-	dd count=1 bs=512 if=/dev/urandom of="$(DESTDIR)/usr/lib/hummingbird/random.seed"
+	mkdir -p "$(DESTDIR)/lib/hummingbird"
+	dd count=1 bs=512 if=/dev/urandom of="$(DESTDIR)/lib/hummingbird/random.seed"

-	$(INSTALL) hummingbird "$(DESTDIR)/usr/bin/hummingbird"
-	$(INSTALL) bin/shutdown "$(DESTDIR)/usr/bin/shutdown"
-	$(INSTALL) bin/reboot "$(DESTDIR)/usr/bin/reboot"
+	$(INSTALL) hummingbird "$(DESTDIR)/bin/hummingbird"
+	$(INSTALL) bin/shutdown "$(DESTDIR)/bin/shutdown"
+	$(INSTALL) bin/reboot "$(DESTDIR)/bin/reboot"

-	$(INSTALL) usr/lib/hummingbird/fs "$(DESTDIR)/usr/lib/hummingbird/fs"
-	$(INSTALL) usr/lib/hummingbird/tty "$(DESTDIR)/usr/lib/hummingbird/tty"
-	$(INSTALL) usr/lib/hummingbird/shutdown "$(DESTDIR)/usr/lib/hummingbird/shutdown"
+	$(INSTALL) usr/lib/hummingbird/fs "$(DESTDIR)/lib/hummingbird/fs"
+	$(INSTALL) usr/lib/hummingbird/tty "$(DESTDIR)/lib/hummingbird/tty"
+	$(INSTALL) usr/lib/hummingbird/shutdown "$(DESTDIR)/lib/hummingbird/shutdown"

 clean:
 	rm hummingbird
